war/src/main/java/io/lumeer/engine/controller/ProjectFacade.java

```diff
@@ -50,68 +66,404 @@ public void setCurrentProjectId(final String projectId) {
       this.projectId = projectId;
    }
 
-   public Map<String, String> getProjectsMap(final String organizationName) {
-      return null;
+   /**
+    * Reads all projects in an organisation that is specified as a parameter
+    *
+    * @param organizationId
+    *       name of the organization
+    * @return map(id, name) values of all projects in an organization
+    */
+   public Map<String, String> readProjectsMap(final String organizationId) {
+      List<DataDocument> documents = dataStorage.searchIncludeAttrs(Project.COLLECTION_NAME, dataStorageDialect.fieldValueFilter(Project.ATTR_ORGANIZATION_ID, organizationId), Arrays.asList(Project.ATTR_PROJECT_NAME, Project.ATTR_PROJECT_ID));
+      Map<String, String> projectsMap = new HashMap<>();
+      documents.forEach(d -> projectsMap.put(d.getString(Project.ATTR_PROJECT_ID), d.getString(Project.ATTR_PROJECT_NAME)));
+      return projectsMap;
    }
 
-   public String getProjectId(final String projectName) {
-      return null;
+   /**
+    * Reads the project id according to its name
+    *
+    * @param projectName
+    *       name of the project
+    * @return id of the project
+    */
+   public String readProjectId(final String projectName) {
+      DataDocument document = dataStorage.readDocumentIncludeAttrs(Project.COLLECTION_NAME, projectNameFilter(projectName), Collections.singletonList(Project.ATTR_PROJECT_ID));
+      return document != null ? document.getString(Project.ATTR_PROJECT_ID) : null;
```

Comment:
Why not to use the other version of `getString` method which does the same thing?

```java
return document.getString(Project.ATTR_PROJECT_ID, null);
```

This can be used in many methods of `ProjectFacade` class.